project/project.go

```diff
@@ -183,11 +180,45 @@ func (p *Project) load(file string, bytes []byte) error {
 		if err != nil {
 			return err
 		}
+		networkNames = append(networkNames, name)
+	}
+
+	for name, config := range serviceConfigs {
+		// Handle NetworkMode
+		// FIXME(vdemeester) not sure it's enough, this might need some rework (here, in service and container)
+		if config.NetworkMode == "" {
+			if p.isNetworkEnabled() {
+				config.NetworkMode = networkNames[0]
```

Comment:
@joshwget oh you are right, I should check if `networkNames` is empty before doing that :sweat: